<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librarianAccountPage</class>
 <widget class="QDialog" name="librarianAccountPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addBookButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editBookButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBookButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displayBooksButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchByAuthorButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchByBookTitleButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
